awx/ui_next/src/screens/Inventory/InventoryDetail/InventoryDetail.jsx

```diff
@@ -76,9 +76,11 @@ function InventoryDetail({ inventory, i18n }) {
           value={
             <ChipGroup numChips={5}>
               {instanceGroups.map(ig => (
-                <Chip key={ig.id} isReadOnly>
-                  {ig.name}
-                </Chip>
+                <div css="padding-right: 5px" key={ig.id} isReadOnly>
```

Comment:
maybe make this ChipLinkContainer and define as a styled-component at the top of the component.